common/remote/rpc/rpc_client.go

```diff
@@ -505,6 +505,9 @@ func (r *RpcClient) Request(request rpc_request.IRequest, timeoutMills int64) (r
 			currentErr = waitReconnect(timeoutMills, &retryTimes, request, errors.New(response.GetMessage()))
 			continue
 		}
+		if response != nil && !response.IsSuccess() {
+			logger.Warnf("%s request received fail response, error code %d, result code %d, message %s", request.GetRequestType(), response.GetErrorCode(), response.GetResultCode(), response.GetMessage())
```

Comment:
modify the print format, the parameter with =or: link, such as: `errCode=%d, resultCode=%d`